### PR TITLE
Add banner with survey link

### LINKED ISSUE #1153: Add persistent banner linking to individual learner survey
It would be great if we had a persistent link to the Individual learn survey: https://learn.wordpress.org/individual-learner-survey/

I'm thinking something like this (mocked up in the inspector):

Screenshot 2022-12-05 at 1 31 31 PM

That would be on every page of the site, including the home page.

### PR DESCRIPTION
**Pull request by hlashbrooke**
This PR adds a persistent banner below the primary site navigation that includes a link to the individual learner survey. Fixes #1153

### CODE REVIEW

**Comment by hlashbrooke** [comment]
I added a new SCSS file for this and I think I built everything correctly from there, but let me know if not.

File: wp-content/themes/pub/wporg-learn-2020/css/components/_notice.scss [hunk 1]
```diff
@@ -0,0 +1,48 @@
+.notice {
```
Comment (adamwoodnz) [inline wp-content/themes/pub/wporg-learn-2020/css/components/_notice.scss:1]: Most of this came from `_locale_notice.scss`, only the survey notice style is new

File: wp-content/plugins/wporg-learn/inc/locale.php [hunk 2]
```diff
@@ -60,8 +60,8 @@ function update_locale_data() {
  * Register style and script assets for later enqueueing.
  */
 function register_assets() {
-	// Locale switcher script.
-	wp_register_script(
```
Comment (adamwoodnz) [inline wp-content/plugins/wporg-learn/inc/locale.php:64]: This meant the script was never loaded on the frontend

**Comment by adamwoodnz** [comment]
Screen Shot 2022-12-05 at 4 55 52 PM

Screen Shot 2022-12-05 at 4 54 49 PM

Screen Shot 2022-12-05 at 4 55 02 PM

**Comment by hlashbrooke** [comment]
This LGTM :shipit:

**Pull request merged**